src/domain/banner/bannerHero/BannerHero.tsx

```diff
@@ -106,20 +117,25 @@ const BannerHero: React.FC<Props> = ({ banner }) => {
       />
       <div
         className={styles.mobileBackgroundImage}
+        data-testid={testIds.mobileBackgroundImage}
         style={{
           backgroundImage: heroBackgroundImageMobile
             ? `url(${heroBackgroundImageMobile})`
             : 'none',
         }}
       />
-      <div
-        className={styles.image}
-        style={{
-          backgroundImage: heroTopLayerImage
-            ? `url(${heroTopLayerImage})`
-            : 'none',
-        }}
-      />
+      {heroTopLayerImage && (
+        <div
+          className={styles.image}
+          data-testid={testIds.heroTopLayerImage}
+          style={{
+            backgroundImage: `url(${heroTopLayerImage})`,
+            backgroundPosition: `center calc(100% - ${
+              location === 'top' ? 5.5 : 0
+            }rem)`,
```

Comment:
inlining this can be little bit hard to read but not a deal breaker.

Reply:
prettier-ignore added